util.go

```diff
@@ -4,12 +4,19 @@ import (
 	"crypto/tls"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 )
 
 func BreakArgsBySeparator() (left, right []string) {
 	seenSeparator := false
 	for _, param := range os.Args[1:] {
+		if strings.HasPrefix(param, "-watch") ||
+			strings.HasPrefix(param, "--watch") ||
+			strings.HasPrefix(param, "-filter") ||
+			strings.HasPrefix(param, "--filter") {
+			continue
+		}
```

Comment:
It's not obvious to me how this works since surely that just results in the argument to those parameters being passed to SSH? What happens with `--filter foo`?

I guess this is a bigger problem perhaps outside the scope of this PR.